proto/regen/bond/v1alpha1/tx.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package regen.bond.v1alpha1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/bond";
+
+// Msg defines the Msg service.
+service Msg {
+
+  // IssueBond issues a new bond
+  rpc IssueBond(MsgIssueBond) returns (MsgIssueBondResponse);
+
+  // SellBond sells a bond to a new owner
+  rpc SellBond(MsgSellBond) returns (MsgSellBondResponse);
+}
+
+// MsgIssueBond is the Msg/IssueBond request type.
+message MsgIssueBond {
+
+  // emission_denom is the unique id of emission
+  string emission_denom = 1;
+
+  // bond name
+  string name = 2;
+
+  // account address of current holder of the bond
+  string holder = 3;
+
+  // face value
+  string face_value = 4;
+
+  // currency of the face value
+  string face_currency = 5;
+
+  // date of issuance
+  google.protobuf.Timestamp issuance_date = 6 [(gogoproto.stdtime) = true];
+
+  // date of maturity
+  google.protobuf.Timestamp maturity_date = 7 [(gogoproto.stdtime) = true];
+
+  // coupon rate
+  string coupon_rate = 8;
+
+  // coupon frequency
+  string coupon_frequency = 9;
+
+  // project code
+  string project = 10;
+
+  // metadata encoded in base64
+  string metadata = 11;
```

Comment:
```suggestion
  // metadata encoded in base64
  bytes metadata = 11;
```

Reply:
If it's base64 encoded, then it should be string. 
BTW: with proto, we don't need to do base64 encoding, so using bytes directly should work.